### DISCUSSION TITLE
repeated new connection and dns monitor issue

### DISCUSSION THREAD

**Discussion by modigopal** [redisson/redisson#6114] [General]
https://github.com/redisson/redisson/issues/5621
https://github.com/redisson/redisson/issues/5589

I am facing issue "Detected DNS change. Master rediss://rts-serverless-test-lhu2ar.serverless.aps1.cache.amazonaws.com:6379 has changed ip from 10.51.218.64 to 10.51.219.38"

Tried with 3.26.1+ versions.

Can anyone help on this?

**Comment by maranza** [comment]
Are you using commecial version or open source version?

**Comment by mrniko** [comment]
you need to upgrade to 3.27.0 or the latest version
